command_line/sort_reflections.py

```diff
@@ -102,10 +103,10 @@ def run(self):
         reflections = reflections.select(perm)
 
         if options.verbose > 0:
-            print("Head of sorted list " + attr + ":")
+            print("Head of sorted list " + params.key + ":")
             n = min(len(reflections), 10)
             for i in range(n):
-                print(reflections[i][attr])
+                print(reflections[i][params.key])
```

Comment:
`attr` is defined nowhere in the whole file.  Judging by the original commit, which predates the use of PHIL and the DIALS option parser, it seems the intended behaviour is to use `params.key`.

The fact that the broken behaviour wasn't caught in a test is also fixed in this commit.